pinot-common/src/test/java/com/linkedin/pinot/pql/parsers/Pql2CompilerTest.java

```diff
@@ -93,4 +94,18 @@ private void testTopZeroFor(Pql2Compiler compiler, String s, final int expectedT
     Assert.assertTrue(groupBy.isSetTopN());
     Assert.assertEquals(expectedTopN, groupBy.getTopN());
   }
+
+  @Test
+  public void rejectInvalidLexerToken() {
+    assertCompilationFails(new Pql2Compiler(), "select foo from bar where baz ?= 2");
+  }
+
+  @Test
+  public void testCStyleInequalityOperator() {
+    Pql2Compiler compiler = new Pql2Compiler();
+
+    BrokerRequest brokerRequest = compiler.compileToBrokerRequest(
```

Comment:
what happens for <  > and ! =.  if there is space between the characters? 
Also =! should throw parsing error?